api-reference/10 UI Components/dxDataGrid/1 Configuration/onCellPrepared.md

```diff
@@ -17,7 +17,7 @@ Information about the event that caused the function's execution.
 This column's [configuration](/api-reference/10%20UI%20Widgets/dxDataGrid/1%20Configuration/columns '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/').
 
 ##### field(e.columnIndex): Number
-The index of the column to which the cell belongs. For details on indexes, see the [Column and Row Indexes](/concepts/05%20Widgets/DataGrid/15%20Columns/12%20Column%20and%20Row%20Indexes.md '/Documentation/Guide/UI_Components/DataGrid/Columns/Column_and_Row_Indexes/') topic.
+The visible column index described in the following topic: [Column and Row Indexes](/concepts/05%20Widgets/DataGrid/15%20Columns/12%20Column%20and%20Row%20Indexes.md '/Documentation/Guide/UI_Components/DataGrid/Columns/Column_and_Row_Indexes/').
 
```

Comment:
I'm not able to update any lines below, so I would like you to make the following change.

Line 69: Allows tracking a variable and performing actions when it changes.

I want to avoid the -ing form ("tracking", "performing") if possible. Who does tracks a variable or performs ("executes" might be better) these actions, the developer or the end user?

Reply:
It is the developer who does this. I guess that we can rephrase this sentence as 'Allows you to track a variable and execute actions when its value changes.'

Reply:
Sounds good. Please apply your suggestion.